addons/website_form/views/snippets/s_website_form.xml

```diff
@@ -74,6 +74,8 @@
                 <we-button data-custom-field="datetime">Date &amp; Time</we-button>
                 <we-button data-custom-field="binary">File Upload</we-button>
             </we-select>
+            <we-input string="Height" data-step="1" data-select-attribute="" data-unit="rows"
```

Comment:
Should not there be a default value, like the 3 rows you set by default on the snippets ? So that if the user removes the value it goes back to 3 rows instead of adding 0 rows?

Reply:
I now understand why I had the conversion issue with the row unit, it happens when a default value is set without the unit. I have to put my default value as "3 rows" instead of "3". Is it really intended to work that way ? I am not saving units so my default should not have unit in it I think.

Reply:
Yes, it is intended to work that way. You could say 3rem and the rem would be converted to rows if possible. Maybe we could allow the convert function to convert from "" to X for all X though.